README.md

```diff
@@ -22,6 +22,8 @@ connector.port=8080
 connector.username=sputnik
 connector.password=PassWd
 connector.useHttps=false
+connector.verifySsl=false
+connector.tag=sputnik
```

Comment:
Can you add some description for new parameter in README?

Reply:
I need to work on README in next PR. There are lots of options not exposed nor described there.